2/contrib/s2i/run

```diff
@@ -172,21 +172,12 @@ CONTAINER_MEMORY_IN_MB=$((CONTAINER_MEMORY_IN_BYTES/2**20))
 
 # multi-arch check ... if x86_64, pick between 32 and 64 bit
 if [[ "$(uname -m)" == "x86_64" ]]; then
-    # Set the JVM architecture used.  Follow OPENSHIFT_JENKINS_JVM_ARCH if set.  If
-    # not, use 32 bit JVM for space efficiency if container size < 2GiB
-    if [[ "${OPENSHIFT_JENKINS_JVM_ARCH}" == "x86_64" || \
-	      ( "${OPENSHIFT_JENKINS_JVM_ARCH}" == "" && "${CONTAINER_MEMORY_IN_MB}" -ge 2048 ) ]]; then
 	alternatives --set java $(alternatives --display java | awk '/family.*x86_64/ { print $1; }')
 	alternatives --set javac $(alternatives --display javac | awk '/family.*x86_64/ { print $1; }')
-    else
-	alternatives --set java $(alternatives --display java | awk '/family.*i386/ { print $1; }')
-	alternatives --set javac $(alternatives --display javac | awk '/family.*i386/ { print $1; }')
-	export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
-    fi
 #set JVM for all other archs
 else
-        alternatives --set java $(alternatives --display java | awk '/family.*'$(uname -m)'/ { print $1; }')
-        alternatives --set javac $(alternatives --display javac | awk '/family.*'$(uname -m)'/ { print $1; }')
+  alternatives --set java $(alternatives --display java | awk '/family.*'$(uname -m)'/ { print $1; }')
+  alternatives --set javac $(alternatives --display javac | awk '/family.*'$(uname -m)'/ { print $1; }')
```

Comment:
I didn't remember off hand how the ppc stuff worked, and this seemed the safest change in that regard, just in case.

If that in fact is not a concern, sure, nuke all the alternative calls

Reply:
can ask @jeffdyoung but i would think we're just going to have two cases:

1) ppc image - the yum install installs the ppc jdk package, invoking java uses it since it's the only "alternative"
2) x86 image - the yum install installs the x86 jdk package, invoking java uses it since it's the only "alternative"

Reply:
I'll get the change ready but not push it until
- let the current test jobs finish to get a base line
- give @jeffdyoung at least some time to respond

Reply:
@bparees @gabemontero 

I think (for some reason) jenkins needed alternatives set? I recall breaking this at one point on the x86 builds.

If it works on x86 w/o setting, then power (and other archs) should be good.

If you do end up having to set alternatives just do this:

```
alternatives --set java $(alternatives --display java | awk '/family.*'$(uname -m)'/ { print $1; }')
alternatives --set javac $(alternatives --display javac | awk '/family.*'$(uname -m)'/ { print $1; }')
```

^ that will work for x86_64, ppc64le.. etc

Reply:
ok thanks @jeffdyoung 

I'll push it up momentarily and we'll see ... this PRs current form just passed e2e-aws-jenkins, so there is the baseline I noted a few minutes ago